LICENSES/GPL-3.0-or-later.txt

```diff
@@ -0,0 +1 @@
+../LICENSE
```

Comment:
I really wish the GitHub UI would make it easier to see whether this file is a symlink or not. (It does show in the file tree on the left, but I don't think it can be seen from the diff list on the right...)

There were multiple cases of PRs in c.g for new modules where authors actually added a text file with this content instead of a symlink... When the tree on the left didn't yet exist it was only possible to distinguish this from the errors the sanity tests emitted ;-)